build-tools/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersAware.java

```diff
@@ -24,11 +25,10 @@ default void useCluster(ElasticsearchCluster cluster) {
             throw new TestClustersException("Task " + getPath() + " can't use test cluster from" + " another project " + cluster);
         }
 
-        cluster.getNodes()
-            .stream()
-            .flatMap(node -> node.getDistributions().stream())
-            .forEach(distro -> dependsOn(getProject().provider(() -> distro.maybeFreeze())));
-        cluster.getNodes().forEach(node -> dependsOn((Callable<Collection<Configuration>>) node::getPluginAndModuleConfigurations));
+        cluster.getNodes().all(node -> node.getDistributions().stream()
+                .forEach(distro -> dependsOn(getProject().provider(() -> distro.maybeFreeze()))));
+        cluster.getNodes().all(node -> dependsOn((Callable<Collection<Configuration>>) node::getPluginAndModuleConfigurations));
+        cluster.getNodes().all(node -> dependsOn((Callable<Collection<Configuration>>) node::getPluginAndModuleConfigurations));
```

Comment:
These two lines are identical. Is it intentional?

Reply:
Yep, looks like a duplication here.